tests/test_runs_gc_perf.py

```
@@ -0,0 +1,115 @@
+            start_time = time.time()
+            runs_fast = runs_gc.discover_all_runs(compute_size=False)
+            time_fast = time.time() - start_time
+
+            print(f"\nTime with compute_size=True: {time_full:.4f}s")
+            print(f"Time with compute_size=False: {time_fast:.4f}s")
+
+            # Assert fast is faster (it should be, significantly)
+            # We use a relaxed assertion because file system caching might affect things,
+            # but usually the difference is large enough.
+            # With 100 runs * 10 files, stat calls add up.
+            self.assertLess(time_fast, time_full)
```

Ruff (0.15.0)

[warning] Use a regular `assert` instead of unittest-style `assertLess`

Replace `assertLess(...)` with `assert ...`

(PT009)